keps/sig-cli/859-kubectl-headers/README.md

```diff
@@ -320,12 +320,21 @@ can be no version skew.
 
 * **How can a rollout fail? Can it impact already running workloads?**
 
-  This feature only affects kubectl; not any cluster components. So it would not
-  be possible to impact running workloads.
+  A danger in the rollout of this feature is adding too much data (too many
+  headers) to each of the API Server calls. We intend to mitigate this risk
+  by defining a MAX_HEADERS concept to ensure the headers to not grow above a
+  certain size. MAX_HEADERS will provide a mechanism to ensure the headers data
+  does not grow without bound. As far as cluster workloads, this feature only
+  affects kubectl; not any cluster components. So it would not be possible to
+  impact running workloads.
 
 * **What specific metrics should inform a rollback?**
 
-  Not applicable.
+  We will measure the **headers-added-round-trip-time** and compare it to the round-trip
```

Comment:
I don't think this will be necessary.  I was more thinking you could add a low cardinality, safe metric to get a sense of how your plan works end to end.